src/components/BookItem/BookItem.js

```diff
@@ -4,7 +4,6 @@ import {Link} from 'react-router-dom';
 
 import reactJPG from '../../images/fullstackreact.jpg';
 import reactSVG from '../../images/react.svg';
-import * as ROUTES from '../../constants/routes';
 
```

Comment:
There would be a conflict about importing 2 .scss files. Just keep the origin one. Actually there is nothing added in this PR, so I don't know why that could be a conflict.